src/Symfony/Component/Translation/Tests/Writer/TranslationWriterTest.php

```diff
@@ -55,9 +55,12 @@ public function testUnwritableDirectory()
         $writer = new TranslationWriter();
         $writer->addDumper('foo', $this->createMock(DumperInterface::class));
 
+        $path = tempnam(sys_get_temp_dir(), '');
+        file_put_contents($path, '');
```

Comment:
we use a similar approach in `FilesystemTest::testMkdirCreatesDirectoriesFails()`